docs/sources/tempo/configuration/manifest.md

```diff
@@ -1103,4 +1103,6 @@ backend_worker:
         instance_addr: ""
         enable_inet6: false
         wait_active_instance_timeout: 10m0s
+    finish_on_shutdown: false
+    job_finish_timeout: 30s
```

Comment:
Is this timeout used for anything other than shutdown? Else I might call it `finish_on_shutdown_timeout`.

Or maybe 1 setting can accomplish both?  if timeout==0 then shutdown immediately, else wait for the timeout.